src/misclibs.F

```diff
@@ -3899,17 +3899,16 @@ subroutine calcmass(nstat,rstat,ruh,rvh,rmh,rho)
 
       tmass=0.0d0 
       !$omp parallel do default(shared) private(i,j,k)
-      !$acc parallel default(present) private(i,j,k) reduction(+:foot,tmass)
-      !$acc loop gang reduction(+:tmass)
+      !$acc parallel loop default(present) collapse(3) reduction(+:tmass)
```

Comment:
Thanks for catching this and it looks fine to collapse the three nested loops. I would suggest adding `gang vector` clause after the `loop` directive for consistency.

Reply:
Thanks for the catch.